src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  color: Color;
+
+  sides: number[];
+
+  constructor(
+    color: Color,
+    sideA: number,
+    sideB: number,
+    sideC: number,
```

Comment:
Whenever you specify public or private on a constructor parameter a corresponding public/private variable is created on the class and filled with the value of the parameter.

```suggestion
    public color: Color,
    public sideA: number,
    public sideB: number,
    public sideC: number,
```